src/main/resources/assets/tfc/models/block/plant/duckweed.json

```diff
@@ -1,7 +1,6 @@
 {
```

Comment:
This is an error: just removing the autogeneration marker does not mean that file stops getting autogenerated. This should've been fixed in resource generation, otherwise if someone's being careless (like I just almost was), this would've been overwritten by the next resource generation)

cc @eerussianguy